### DISCUSSION TITLE
wrapper -> plugins

### DISCUSSION THREAD

**Discussion by ibrahimali-ca** [connery-io/connery-sdk#72] [General]
why did this project pivot from an open source wrapper to plugin platform?

**Comment by machulav** [comment]
We have not pivoted the project. We just changed the terminology from connector to plugin as we found that plugins are more understandable for people. We also slightly adjusted the project description because of the same reason. However, project goals remain the same as before, and the project remains open-source as well. 

You can find more info about our goals and open-source position in the repository's README.
